platform/windows/display_server_windows.cpp

```diff
@@ -3959,10 +3959,18 @@ DisplayServer::WindowID DisplayServerWindows::_create_window(WindowMode p_mode,
 
 		wd.im_position = Vector2();
 
-		// FIXME this is wrong in cases where the window coordinates were changed due to full screen mode; use WindowRect
-		wd.last_pos = p_rect.position;
-		wd.width = p_rect.size.width;
-		wd.height = p_rect.size.height;
+		if (p_mode == WINDOW_MODE_FULLSCREEN || p_mode == WINDOW_MODE_EXCLUSIVE_FULLSCREEN || p_mode == WINDOW_MODE_MAXIMIZED) {
+			RECT r;
+			if (GetWindowRect(wd.hWnd, &r)) {
+				wd.last_pos = Point2(r.left, r.top);
+				wd.width = r.right - r.left;
+				wd.height = r.bottom - r.top;
+			}
```

Comment:
This should be client rect, with the position adjusted to account for the different screen coordinate system used by Godot and Windows.

```suggestion
			GetClientRect(wd.hWnd, &r);
			ClientToScreen(wd.hWnd, (POINT *)&r.left);
			ClientToScreen(wd.hWnd, (POINT *)&r.right);
			wd.last_pos = Point2i(rect.left, rect.top) - _get_screens_origin();
			wd.width = r.right;
			wd.height = r.bottom;
```

Reply:
Thank you for the correction. I just replaced `rect.left` by `r.left`.